pkg/osutil/osutil.go

```diff
@@ -115,6 +123,14 @@ func CopyDir(
 	return nil
 }
 
+func validateAllVariablesSubstituted(s string) error {
```

Comment:
💡 Funct name is `validate...` validation suggest it will return bool, or if I can recommend why not say `checkAllVariablesSubstituted` ?

Reply:
sounds good- updated